src/bindings/python/src/openvino/preprocess/tokenization/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .convert_tokenizer import convert_tokenizer
```

Comment:
would we like to have separate API for tokenizers?
Why can't we or shouldn't we add it into opset to make models OOB converted?